static/js/ducks/source.js

```diff
@@ -102,6 +104,10 @@ export function fetchSource(id) {
   );
 }
 
+export function fetchColorMag(id) {
```

Comment:
Why keep this if it's never used? Same for action type above

Reply:
Yes you are right, I forgot about that.